modules/geolocation/unittests.py

```diff
@@ -202,6 +202,100 @@ def test_point_set_2(self):
         actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+        
+        
+class TestOutputAnalysis(unittest.TestCase):
+    """
+    Tests Geolocation.outputanalysis(inputLocationTupleList)
+
+    """
+    def test_all_ones(self):
+        outputAnalysis = geolocation.OutputAnalysis()
+
+    # Setup
+        outputList = np.array([[[1, 1], [1], [1]]])
+
+        expectedCoordPair= (np.array([[1],  [1]]))
+        expectedError= (np.array([1]))
+
+        # Run
+        actual = outputAnalysis.get_best_location(outputList)
+
+        np.testing.assert_equal(actual[0], expectedCoordPair)
```

Comment:
Because we are working with floating point numbers, use `assert_almost_equal()` instead.